src/services/baseService.ts

```diff
@@ -83,4 +84,12 @@ export abstract class BaseService {
         }));
     });
   }
+
+  protected getServerlessConfig(): ServerlessConfig {
+    return this.serverless.utils.readFileSync("serverless.yml") as ServerlessConfig;
```

Comment:
There is probably an easier way to do this with the Serverless Utils, but I haven't dug deep enough yet

Reply:
Where is this used? What part of the config is required?  Most of this is just on the `serverless` object itself, no?

Reply:
Need to investigate further here. With the new debugging tools, that should be a lot easier :)